applications/ConvectionDiffusionApplication/custom_elements/symbolic_dynamic_eulerian_convection_diffusion_explicit.h

```diff
@@ -0,0 +1,317 @@
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class SymbolicDynamicEulerianConvectionDiffusionExplicit
+ * @ingroup ConvectionDiffusionApplication
+ * @brief This element solves the convection-diffusion equation, stabilized with
+ * algebraic subgrid scale or orthogonal subgrid scale.
+ * @details This element solves the convection-diffusion equation:
+ * $ \frac{\partial \phi}{\partial t} + v \cdot  \nabla \phi + \phi \nabla \cdot v - \nabla \cdot k \nabla \phi = f $
+ * where $ \phi $ is the scalar unknown, $ v $ the convective velocity, $ k > 0 $ the diffusivity coefficient,
+ * $ f $ the forcing term.
+ * Dynamic algebraic subgrid scale and dynamic orthogonal subgrid scale methods are exploited for stabilization.
+ * The element is designed to use an explicit integration method.
+ * @author Riccardo Tosi
+ */
+template< unsigned int TDim, unsigned int TNumNodes = TDim + 1>
+class SymbolicDynamicEulerianConvectionDiffusionExplicit : public SymbolicQuasiStaticEulerianConvectionDiffusionExplicit<TDim,TNumNodes>
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+        typedef SymbolicQuasiStaticEulerianConvectionDiffusionExplicit<TDim,TNumNodes> BaseType;
+        typedef typename BaseType::ElementVariables ElementVariables;
+        typedef Node < 3 > NodeType;
+        typedef Geometry<NodeType> GeometryType;
+        typedef Geometry<NodeType>::PointsArrayType NodesArrayType;
+        typedef Vector VectorType;
+        typedef Matrix MatrixType;
+        typedef std::size_t IndexType;
+        typedef std::vector<std::size_t> EquationIdVectorType;
+        typedef std::vector< Dof<double>::Pointer > DofsVectorType;
+        typedef GeometryData::IntegrationMethod IntegrationMethod;
+
+    /// Pointer definition of SymbolicDynamicEulerianConvectionDiffusionExplicit
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(SymbolicDynamicEulerianConvectionDiffusionExplicit);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    //Constructors.
+
+    /// Default constuctor.
+    SymbolicDynamicEulerianConvectionDiffusionExplicit(IndexType NewId, GeometryType::Pointer pGeometry);
+    SymbolicDynamicEulerianConvectionDiffusionExplicit(IndexType NewId, GeometryType::Pointer pGeometry, Properties::Pointer pProperties);
+
+    /// Destructor.
+    virtual ~SymbolicDynamicEulerianConvectionDiffusionExplicit();
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    Element::Pointer Create(
+        IndexType NewId,
+        NodesArrayType const& ThisNodes,
+        Properties::Pointer pProperties) const override;
+
+    Element::Pointer Create(
+        IndexType NewId,
+        GeometryType::Pointer pGeom,
+        Properties::Pointer pProperties) const override;
+
+    void CalculateLocalSystem(
+        MatrixType& rLeftHandSideMatrix,
+        VectorType& rRightHandSideVector,
+        ProcessInfo& rCurrentProcessInfo) override;
```

Comment:
Please use the new interfaces that have a const ProcessInfo 

See the baseclass or the template

I will make an issue abt this soon

Reply:
Ok, done.